pkg/sfu/downtrack.go

```diff
@@ -744,6 +748,12 @@ func (d *DownTrack) Codec() webrtc.RTPCodecCapability {
 	return d.codec
 }
 
+func (d *DownTrack) Mime() mime.MimeType {
+	d.bindLock.Lock()
+	defer d.bindLock.Unlock()
+	return d.mime
```

Comment:
mime could change if the upstream codec changed (codec regression)

Reply:
:phew: good catch, will take a look. Also, need to update `connectionstats` module with mime as the mime is reported to analytics.

Reply:
Addressed in this [commit](https://github.com/livekit/livekit/pull/3407/commits/faea596f0eb3cd6aa7e6ccb3c658e34abd54e16e) @cnderrauber . Please have a look when you get a chance.

@anunaym14 @davidzhao Will need an approval from one of you two as three of us have commits in this PR. Rubber stamp should be fine I think as @paulwe and @cnderrauber have looked at it and made commits to correct some stuff.